code/modules/mob/living/carbon/carbon.dm

```diff
@@ -785,6 +785,11 @@ var/list/ventcrawl_machinery = list(/obj/machinery/atmospherics/unary/vent_pump,
 			if(!buckled)
 				return
 			buckled.user_unbuckle_mob(src,src)
+			if(istype(I, /obj/item/restraints/handcuffs/cable))
+				playsound(loc, 'sound/effects/snap.ogg', 50, 1, -1)
+				visible_message("<span class='danger'>As [src] manages to unbuckle, It destroys [I] as well!</span>")
```

Comment:
Creo que en la mayoría de los textos evitan las comas
Qué tal algo como:
[src] manages to unbuckle destroying [I] in the process!

Reply:
Vale vale